### ISSUE TITLE
Feature request: App Icon chooser

### ISSUE THREAD

**Issue by rklie** [Ranchero-Software/NetNewsWire#4058]
Hi!

I've been using your app for a while now, and I absolutely love it. Best RSS reader I've used in my life.

One thing I want to suggest is the ability to choose the app icon. I know it's purely cosmetics, but would love for a simple selector of different app icons, could even be same design but different colors. I think people would love that flexibility.

Keep up the good work!!

**Comment by on-com** [comment]
Late, and this might not be what you're looking for, but you can change icons in general through File > Get Info, then dragging or pasting an image onto the file's icon at the top right. Changes typically don't appear until you relaunch the dock.

<img width="530" height="1232" alt="Image" src="https://github.com/user-attachments/assets/d02673a1-f7f6-4a03-80ea-598589d32e11" />

**Comment by rklie** [comment]
@on-com Yeah, that works for the Mac app. I was mostly referring to mobile. Updated my original request to reflect that.

Thanks for the suggestion though!
